types/meteor/globals/accounts-base.d.ts

```diff
@@ -146,3 +146,23 @@ declare module Accounts {
 declare module Accounts {
     function onLogout(func: (user: Meteor.User, connection: Meteor.Connection) => void): void;
 }
+
+declare module Accounts {
+    interface LoginMethodOptions {
+        methodName?: string;
+        methodArguments?: any[],
+        validateResult?: Function,
+        userCallback?: (err: any) => void
+    }
+
+    type Password = string | {
+        digest: string;
+        algorithm: "sha-256"
+    }
+
+    function callLoginMethod(options: LoginMethodOptions): void;
+
+    function registerLoginHandler(name: string, handler: Function): void;
```

Comment:
There is a very good description of those methods available in the original JS files. It would be great if this description is also added here (with /** ... */ style comments) so that it is shown in code completion hints. Also, I think handler definition could be more strict?

Reply:
Absolutely - I wasn’t sure how much typings you preferred (I like lots of types)

Reply:
Seems I also made the change in the wrong file (in "globals")

Reply:
OK I think I got a better version now. It seems like someone previously hadn’t run scripts/generate-globals because the files in globals picked up some changes I hadn’t done